config/initializers/constants.rb

```diff
@@ -0,0 +1 @@
+USERS_LIMIT = 4
```

Comment:
how do you decide whether to put a constant in constants.rb or in some model?

Reply:
Sir, USERS_LIMIT is generic and can be used anywhere in the application from Plans description to FAQs.